drivers/wifi/esp_hosted/esp_hosted_hal.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2025 Arduino SA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef MICROPY_INCLUDED_DRIVERS_ESP_HOSTED_HAL_H
```

Comment:
Is the prefix `MICROPY` expected here?

Reply:
No, sorry. I started with the same hal header from the MicroPython driver (which I also wrote).